plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -120,6 +121,27 @@ def package_headers(filename):
     return headers
 
 
+def package_headers_drpm(filename):
```

Comment:
It does partly. Method `` def package_headers(filename):`` extracts RPM header from RPMs/SRPMs/DRPMs, but from design of format in DRPM a RPM header of the new package is present, which is what the previous metioned does, it extracts data about the new RPM package from DRPM, but to obtain information about the old package and sequence we need to extract DRPM header as well.